cupy/cusolver.pyx

```diff
@@ -65,14 +65,14 @@ _available_cuda_version = {
 }
 
 _available_hip_version = {
-    'gesvdjBatched': (309, None),  # = rocsolver_<t>gesvd_batched
-    'potrfBatched': (306, None),
     # Below are APIs supported by CUDA but not yet by HIP. We need them here
     # so that our test suite can cover both platforms.
```

Comment:
Is the comment outdated?

Reply:
Good catch @toslunar! I just updated it to align with that in `cupy/cusparse.py`.